### PR TITLE
Add mfaType and conversions of string -> number

### PR DESCRIPTION
**Pull request by VDubber**
# Description

Thank you for contributing to a JupiterOne integration!

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.

## Summary

Add mfaEnabled to user
Add mfaType to user
convert masterPasswordStrength to number
convert securityScore to number

## Type of change

Please leave any irrelevant options unchecked.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### General Development Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### Integration Development Checklist:

Please leave any irrelevant options unchecked.

- [ ] I have checked for additional permissions required to call any new API
      endpoints, and have documented any additional permissions in
      `jupiterone.md`, where necessary.
- [ ] My changes properly paginate the target service provider's API
- [ ] My changes properly handle rate limiting of the target service provider's
      API
- [ ] My new integration step is instrumented to execute in the correct order
      using `dependsOn`
- [ ] I have referred to the
      [JupiterOne data model](https://github.com/JupiterOne/data-model/tree/main/src/schemas)
      to ensure that any new entities/relationships, and relevant properties,
      match the recommended model for this class of data
- [ ] I have updated the `CHANGELOG.md` file to describe my changes
- [ ] When changes include modifications to existing graph data ingestion, I've
      reviewed all existing managed questions referencing the entities,
      relationships, and their property names, to ensure those questions still
      function with my changes.

### CODE REVIEW

File: src/steps/users/converter.ts [hunk 1]
```diff
@@ -32,16 +33,20 @@ export function createUserEntity(user: User): Entity {
         lastLogin: parseTimePropertyValue(user.last_login),
         email: user.username,
         active: !user.disabled,
+        ...(user.multifactor && {
+          mfaEnabled: true,
+        }),
```
Comment (zemberdotnet) [inline src/steps/users/converter.ts:38]: Is this just setting multifactor? I'm a little unfamiliar with the syntax here.

File: src/steps/users/converter.ts [hunk 2]
```diff
@@ -32,16 +33,20 @@ export function createUserEntity(user: User): Entity {
...
+        mfaType: user.multifactor,
         neverLoggedIn: user.neverloggedin,
         admin: user.admin,
         shortLoginId,
         emailDomain: [emailDomain],
         webLink: `https://admin.lastpass.com/users/view?users=${searchParamForWebLink}`,
 
         duoUsername: user.duousername,
-        masterPasswordStrength: user.mpstrength,
+        masterPasswordStrength: parseStringPropertyValue(user.mpstrength),
```
Comment (zemberdotnet) [inline src/steps/users/converter.ts:47]: 🥳

File: src/steps/users/converter.ts [hunk 3]
```diff
@@ -32,16 +33,20 @@ export function createUserEntity(user: User): Entity {
...
         passwordResetRequired: user.password_reset_required,
-        securityScore: user.totalscore,
+        securityScore: parseStringPropertyValue(user.totalscore),
         linkedAccount: user.linked,
         sitesCount: user.sites,
         notesCount: user.notes,
```
Comment (adam-in-ict) [inline src/steps/users/converter.ts:52]: Should we go ahead and assume other properties with the word `Count` in them (`sitesCount`, `notesCount`, `applicationCount`, `attachmentsCount`) should also be numbers?

**Review by austinkelleher — APPROVED** [review]

File: src/steps/users/converter.ts (hunk 1 above)
Comment (VDubber) [reply src/steps/users/converter.ts:38]: It results to: if multifactor is truthy, include mfaEnabled: true in the parent object.

File: src/steps/users/converter.ts (hunk 3 above)
Comment (VDubber) [reply src/steps/users/converter.ts:52]: Looking at the data, they already come back as numbers. 👍

**Pull request merged**